hack/lib/golang.sh

```diff
@@ -433,6 +433,50 @@ kube::golang::get_physmem() {
   echo 1
 }
 
+kube::golang::unhide_godeps() {
+  echo "Restoring Godeps to their rightful location"
+  mv "${KUBE_ROOT}/.Godeps" "${KUBE_ROOT}/Godeps/"
+  rm "${KUBE_ROOT}/_output/local/go/vendor"
+}
+
+kube::golang::hide_godeps() {
+  echo "Hiding godeps from GB"
+  mv "${KUBE_ROOT}/Godeps" "${KUBE_ROOT}/.Godeps"
+  ln -s "${KUBE_ROOT}/.Godeps/_workspace" -T "${KUBE_ROOT}/_output/local/go/vendor" >/dev/null 2>&1 || true
```

Comment:
updated diff pushed. When I set things up by hand I wanted to hide Godeps since gb would run our Godeps directory since by default it wants to use "...".   Since gb (because of filepath.Walk) won't follow symlinks it actually can't use our _output/local/go/src/k8s.io/kubernetes symlink without the explicit pkgs to build.  And, since we are giving it explicit packages we don't need to worry about it walking Godeps and building those. (and they are all going to get build in vendor anyway, so no matter what it doesn't matter)